### ISSUE TITLE
Live UI: seeing loading state when bid has already been sent 

### ISSUE THREAD

**Issue by katarinabatina** [artsy/eigen#1681]
![image](https://cloud.githubusercontent.com/assets/6482926/15974318/dac998a0-2f15-11e6-9ee4-66271ab9a6dc.png)

**Comment by devangt** [comment]
This one is critical and seems like we regression from behavior we have seen up until now. So we recommend keeping this. 

The true issue is that the bidding state gets gets stuck even after operator has approved the bid as opposed to telling me that I am highest bidder. 

Lets root cause.

**Comment by craigspaeth** [comment]
And more importantly even after further floor bids it was stuck in Bidding... state—the fact that it get stuck makes it a show stopper.

**Comment by craigspaeth** [comment]
1. Open a new lot
2. Tap bid in Eigen
3. Accept bid in operator (Eigen says bid... it should say "highest bidder"... wait for the critical part)
4. Tap bid again in Eigen (Bid doesn't show up in operator)
5. In operator hit Floor bid (Notice bid button unlocks in Eigen)
6. Tap bid again... **bid doesn't show up in operator** < Real show stopper—should be fixed in step 3 though

**Comment by ashfurrow** [comment]
The sellingToUser returned by causality is null in step 3, this is fixed by a pending Causality PR. Unsure if the bug will remain once causality is fixed, need to test afterward.

**Comment by craigspaeth** [comment]
Thanks Ash! I was just about to say hold on... this might not be Eigen-side :stuck

**Comment by craigspaeth** [comment]
Ok this is actually a simple MP thing. Closing in favor of https://github.com/artsy/metaphysics/issues/312

**Comment by ashfurrow** [comment]
I'm not seeing the connection, can you clarify when you ha e a chance?

**Comment by devangt** [comment]
I believe Craig shared with you - but putting here as well.
- When a user got to more than 10 auctions registered for we were not getting a registered status correctly at the metaphysics layer
- this led to the user ending up in a state that that was somewhere between observer and approve bidder.
- so user felt that the device was being able to send bids as we got all events back but the sending bid failed. (ideally bid button in this sate should have been disabled as we confirmed that no bids were making it through)
- All the events in the stream were coming through.
